client/lib/push-notifications/service-worker.js

```diff
@@ -6,12 +6,24 @@
 'use strict';
 /* eslint-enable */
 
+var queuedMessages = [];
+
+self.addEventListener( 'install', function( event ) {
+	event.waitUntil( self.skipWaiting() );
+} );
+
+self.addEventListener( 'activate', function( event ) {
+	event.waitUntil( self.clients.claim() );
+} );
+
```

Comment:
I didn't understand what these event listeners were doing. Then I found an identical example in https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerGlobalScope/skipWaiting that helped me understand what it was doing. A comment explaining the intent behind these event listeners would help! :-)